styles/styles.css

```diff
@@ -390,8 +390,8 @@ main pre {
   background-position: center;
   background-size: cover;
   display: inline-block;
-  height: 16px;
-  width: 16px;
+  height: 24px;
```

Comment:
The height being 20px as well would work, but, for some reason that I don't fully understand, this also controls the alignment of the svg, if I used 20px the icons would no longer be centred